documentation/projects.md

```diff
@@ -100,6 +103,27 @@ MODULEPATH:
 Use `bolt plan show <plan-name>` to view details and parameters for a specific plan.
 ```
 
+You can also use glob patterns to match multiple plan or task names. For
+example, to surface all tasks and plans in a project named `myproject`, you
+would use the following `bolt-project.yaml` file:
+
+```yaml
+name: myproject
+plans:
+- myproject::*
+tasks:
+- myproject::*
```

Comment:
```suggestion
- "[mno]yproject::*"
```
Starting the regex with special characters can be a little tricky, but wrapping the phrase in a string should always work. Either stating that or having an example seems useful. This also applies for anything that starts with `*`.

Reply:
Added a short paragraph on wrapping patterns that start with a metacharacter.